hubble/Makefile

```diff
@@ -1,31 +1,43 @@
-GO := go
-GO_BUILD_FLAGS =
-GO_TEST_FLAGS =
-GO_BUILD = CGO_ENABLED=0 $(GO) build $(GO_BUILD_FLAGS)
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Authors of Hubble
+
+.DEFAULT_GOAL := all
+
+ROOT_DIR := $(shell dirname "$(realpath $(lastword $(MAKEFILE_LIST)))")
```

Comment:
is this needed? We're not using include directories so relative paths e.g. `../Makefile.defs` are unambiguous.

Reply:
Same as above, updated